lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,68 @@
+# Gerrit
+
+## Authentication
+
+First, ![HTTP access token](/assets/images/gerrit-http-password.png) for the Renovate account.
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
+
+- Set your HTTP access token as a `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--password=`
+
+Make sure this user is allowed to assign the Code-Review label with "+2" to their own changes or "automerge" can't work.
+
+Remember to set `platform=gerrit` somewhere in your Renovate config file.
+
+## Renovate PR/Branch-Model with Gerrit and needed Permissions
+
+If you use the "Code-Review" label and want `automerge` working, then you have to enable `gerritAutoApprove=true` in your Renovate config.
+In this case Renovate will automatically add the _Code-Review_ label with the value "+2" to each created "pull-request" (Gerrit-Change).
+
+!!! note
+The login should be allowed to give +2 for the Code-Review label.
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! note
    The login must be allowed to give +2 for the Code-Review label.
```

Material for MkDocs admonitions follow this pattern:

```markdown
<!-- prettier-ignore -->
!!! type-of-admontion
    Four spaces before the text of the admonition.
```

The `prettier-ignore` line blocks Prettier from removing the four spaces, which in turn messes up the admonition.